src/jit/codegenxarch.cpp

```diff
@@ -1386,13 +1386,13 @@ void CodeGen::genCodeForDivMod(GenTreeOp* treeNode)
         }
         else
 #endif
-        if (dividend->gtRegNum != REG_RAX)
+            if (dividend->gtRegNum != REG_RAX)
         {
             // dividend must be in RAX
             inst_RV_RV(INS_mov, REG_RAX, dividend->gtRegNum, targetType);
         }
 
-        // zero or sign extend rax to rdx
+// zero or sign extend rax to rdx
```

Comment:
Another jit-format contribution - I moved the comment below the if to avoid the problem.

---
In reply to: [79524539](https://github.com/dotnet/coreclr/pull/7252#discussion_r79524539) [](ancestors = 79524539)